packages/cli/src/actions/init.ts

```diff
@@ -1,21 +1,24 @@
 import { join, dirname } from 'path';
 import { execSync as exec } from 'child_process';
-import fse, { symlink, readJson, ensureDir } from 'fs-extra';
+import fse, { readJson, ensureDir, symlink } from 'fs-extra';
 import { prompt } from 'inquirer';
 import { sync } from 'command-exists';
 import * as os from 'os';
 import { ora, Constants } from '../utils';
 
 import { InitCommandHandler } from '../types';
-import { isLocal, optionalNpmClients, daemonPackage, boardPackage } from '../config';
+import { optionalNpmClients, daemonPackage, boardPackage, isLocal } from '../config';
 
 const {
   BOA_CONDA_INDEX,
   BOA_CONDA_MIRROR
 } = Constants;
 
 async function npmInstall(npmClient: string, name: string, beta: boolean, cwd: string, env: NodeJS.ProcessEnv): Promise<void> {
+  exec(`${npmClient} init -f`, { cwd, env, stdio: 'ignore' });
```

Comment:
```suggestion
  exec(`"${npmClient}" init -f`, { cwd, env, stdio: 'ignore' });
```

Reply:
For robustness, the first argument is `The command to run, with space-separated arguments`, if the `npmClient` has a space, the command failed.

Reply:
Look at:
https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback